README.md

```diff
@@ -1,45 +1,33 @@
 # script-runner
 
-[About this repo](#about-this-repo) |  [Commands](#commands-optional) | [How to use this repo](#how-to-use-this-repo)
+[About this repo](#about-this-repo) | [Development](#development) | [Usage](#usage)
 
 ## About this repo
+
 **Introduction:**
 
-This is a simple CLI (`script-runner`) which allows you to run Wasm
-scripts intended for ShopifyVM. Scripts will run using
+This is a simple CLI (`script-runner`) which allows you to run Shopify Functions intended for ShopifyVM. Functions will run using
```

Comment:
How about: 

`This is a simple CLI (`script-runner`) that lets you run Shopify Functions for ShopifyVM.
Functions will run using the provided JSON input file and will output JSON.`

Reply:
I'm not as confident about the 2nd sentence. It seems odd to me.

Reply:
I think maybe `It accepts a JSON input file and outputs the result as JSON` makes more sense. WDYT?

Reply:
👍